docs/Geolocation.md

```diff
@@ -0,0 +1,42 @@
+# Geolocation
+
+- [Summary](#summary)
+- [Examples](#examples)
+- [Origin](#origin)
+- [Future Functionality](#future-functionality)
+
+# Summary
+
+Starting with `v0.14.0`, Brim now adds [geolocation](https://en.wikipedia.org/wiki/Geolocation) fields to the `conn` records
+of Zeek logs that are generated from imported pcaps.
+
+# Examples
+
+The following screenshot shows where the geolocation fields may be found in the Log Detail view:
+
+![Geolocation in Log Detail](media/Geolocation-Log-Detail.png)
+
+This screenshot shows an example aggregation that uses geolocation data:
+
+![Geolocation Aggregation](media/Geolocation-Aggregation.png)
+
+# Origin
+
+The data is added by the [geoip-conn](https://github.com/brimsec/geoip-conn) Zeek package. For details on the origin and accuracy
+of the geolocation data, see the [README](https://github.com/brimsec/geoip-conn/blob/master/README.md).
+
+# Future Functionality
+
+There are additional geolocation features in Brim that may be added in the future, depending on demand from the community.
+The following issues are currently being held open to gather interest:
+
+| **Issue**                                              |**Description**                                 |
+|--------------------------------------------------------|------------------------------------------------|
+| [brim/936](https://github.com/brimsec/brim/issues/936) | GeoLocation map visualization                  |
```

Comment:
Are the dual 'G' & 'L' capitals of `GeoLocation` intended?

Reply:
Fair point. I seem to have gotten into a bad habit of sometimes typing "GeoLocation", which I think was a side effect of typing "GeoLite2", which does, indeed, have the capital "L" in the middle. However, you're right that "geolocation" is not typically written that way, so I've fix it in my issue descriptions/comments and here in this branch. Thanks for flagging!